Sprint-2/debug/2.js

```diff
@@ -10,5 +10,41 @@ console.log(`The last digit of 42 is ${getLastDigit(42)}`);
 console.log(`The last digit of 105 is ${getLastDigit(105)}`);
 console.log(`The last digit of 806 is ${getLastDigit(806)}`);
 
-// This program should tell the user the last digit of each number.
-// Explain why getLastDigit is not working properly - correct the problem
+/* This program should tell the user the last digit of each number.
+ the function doesn’t have a parameter to receive the num variable it’s trying to process. 
```

Comment:
This is true, but not complete. It doesn't have a parameter, so why is it returning 3? Where does the 3 come from?